docs/en/administration/query_profile_details.md

```diff
@@ -0,0 +1,1366 @@
+- Description: Shared buffer details.
+- Level: Secondary Metric
+- Sub-metrics: DirectIOBytes, DirectIOCount, DirectIOTime, SharedIOBytes, SharedIOCount, SharedIOTime
+
+##### DirectIOBytes
+
+- Description: Amount of data directly read by IO.
+- Level: Tertiary Metric
+
+##### DirectIOCount
+
+- Description: Number of times direct IO is performed.
+- Level: Tertiary Metric
+
+##### DirectIOTime
+
+- Description: Time taken for direct IO.
+- Level: Tertiary Metric
+
+##### SharedIOBytes
+
+- Description: Amount of data read by shared IO.
+- Level: Tertiary Metric
+
+##### SharedIOCount
+
+- Description: Number of times shared IO is performed.
+- Level: Tertiary Metric
+
+##### SharedIOTime
+
+- Description: Time taken for shared IO.
+- Level: Tertiary Metric
+
+##### IOTaskWaitTime
+
+- Description: Waiting time from successful submission to scheduled execution of IO tasks.
+- Level: Primary Metric
+
+##### SubmitTaskCount
+
+- Description: Number of times IO tasks are submitted.
+- Level: Primary Metric
+
+##### SubmitTaskTime
+
+- Description: Time taken to submit tasks.
+- Level: Primary Metric
+
+##### PeakIOTasks
+
+- Description: Peak number of IO tasks.
+- Level: Primary Metric
+
+##### PeakScanTaskQueueSize
+
+- Description: Peak size of the IO task queue.
+- Level: Primary Metric
+
+### Exchange Operator
+
+#### Exchange Sink Operator
+
+##### ChannelNum
+
+Description: Number of channels. Generally, the number of channels is equal to the number of receivers.
+
+##### DestFragments
+
+Description: List of destination FragmentInstance IDs.
+
+##### DestID
+
+Description: Destination node ID.
+
+##### PartType
+
+Description: Data distribution mode, including: UNPARTITIONED, RANDOM, HASH_PARTITIONED, and BUCKET_SHUFFLE_HASH_PARTITIONED.
+
+##### SerializeChunkTime
+
+Description: Time taken to serialize chunks.
+
+##### SerializedBytes
+
+Description: Size of serialized data.
+
+##### ShuffleChunkAppendCounter
+
+Description: Number of Chunk Append operations when PartType is HASH_PARTITIONED or BUCKET_SHUFFLE_HASH_PARTITIONED.
+
+##### ShuffleChunkAppendTime
+
+Description: Time taken for Chunk Append operations when PartType is HASH_PARTITIONED or BUCKET_SHUFFLE_HASH_PARTITIONED.
+
+##### ShuffleHashTime
+
+Description: Time taken to calculate hash when PartType is HASH_PARTITIONED or BUCKET_SHUFFLE_HASH_PARTITIONED.
+
+##### RequestSent
+
+Description: Number of data packets sent.
+
+##### RequestUnsent
+
+Description: Number of unsent data packets. This metric is non-zero when there is a short-circuit logic; otherwise, it is zero.
+
+##### BytesSent
+
+Description: Size of sent data.
+
+##### BytesUnsent
+
+Description: Size of unsent data. This metric is non-zero when there is a short-circuit logic; otherwise, it is zero.
+
+##### BytesPassThrough
+
+Description: Size of data not transmitted over the network when the destination node is the current node, i.e., the size of PassThrough data. Enabled by `enable_exchange_pass_through`.
```

Comment:
suggestion: If the destination node is the current node, data will not be transmitted over the network, which is called passthrough data. This metric indicates the size of such passthrough data. Passthrough is controlled by `enable_exchange_pass_through`.